README.md

```diff
@@ -170,7 +170,7 @@ For simple use cases when you have a few terms that don't change very often.
 
 Run the generator to install configuration files and an example authority.
 
-    rails generate qa:local
+    rails generate qa:local:files
```

Comment:
Should we add a new section to the README for table-based authorities? That way we can reinforce what the generator spits out about having to manually create an index.